### ISSUE TITLE
'admob_flutter/admob_flutter-Swift.h' file not found'

### ISSUE THREAD

**Issue by jiharal** [kmcgill88/admob_flutter#54]
=== BUILD TARGET firebase_messaging OF PROJECT Pods WITH CONFIGURATION Debug ===
    /usr/local/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.1/ios/Classes/AdmobFlutterPlugin.m:20:9: fatal error: 'admob_flutter/admob_flutter-Swift.h' file not found
    #import <admob_flutter/admob_flutter-Swift.h>
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 error generated.

**Comment by gomgomize** [comment]
I have same problem with him. Is there any solution?

**Comment by sergey-s-moiseev** [comment]
Try to add in Podfile next values
```
target 'Runner' do
  use_frameworks! # this line
```
and

```
pre_install do |installer| #this function
  installer.analysis_result.specifications.each do |s|
    s.swift_version = '4.2' unless s.swift_version
  end
end

install! 'cocoapods', :disable_input_output_paths => true

post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['ENABLE_BITCODE'] = 'NO'
      config.build_settings['SWIFT_VERSION'] = '4.2' #this line
    end
  end
end
```
It's helps me

**Comment by Xoshbin** [comment]
> Try to add in Podfile next values
> 
> ```
> target 'Runner' do
>   use_frameworks! # this line
> ```
> 
> and
> 
> ```
> pre_install do |installer| #this function
>   installer.analysis_result.specifications.each do |s|
>     s.swift_version = '4.2' unless s.swift_version
>   end
> end
> 
> install! 'cocoapods', :disable_input_output_paths => true
> 
> post_install do |installer|
>   installer.pods_project.targets.each do |target|
>     target.build_configurations.each do |config|
>       config.build_settings['ENABLE_BITCODE'] = 'NO'
>       config.build_settings['SWIFT_VERSION'] = '4.2' #this line
>     end
>   end
> end
> ```
> 
> It's helps me

Didn't work for me!

**Comment by craig006** [comment]
@jiharal @Xoshbin @gomgomize Any of you manage to get past this? I'm also getting it.

**Comment by Xoshbin** [comment]
No, It's not working on iOS

**Comment by craig006** [comment]
Managed to get this working. Seems like this is caused if when first creating your flutter app you used obj-c for the ios side.

I created a new flutter app with swift for ios instead and copied my pupspec, lib and asset folders over and then it worked. I could not find a way to make the old obj project work.

**Comment by sunjianan9900** [comment]
delete ios file
and run 'flutter create  -i swift .'
rebuild in swift

**Comment by XHL-Mao** [comment]
[flutter/flutter#16049 (comment)](https://github.com/flutter/flutter/issues/16049#issuecomment-382629492)

**Comment by calber** [comment]
still not working, any update ?

**Comment by kmcgill88** [comment]
https://github.com/kmcgill88/admob_flutter/issues/54#issuecomment-524782853

https://github.com/kmcgill88/admob_flutter/issues/54#issuecomment-529354951

**Comment by EliasDeuss** [comment]
This has been fixed in #150
